src/Donors/DonorsAdminPage.php

```diff
@@ -135,7 +135,7 @@ function showReactTable() {
                     },
                 })
                     .then((res) => {
-                        window.location.reload();
+                        window.location.href = '/wp-admin/edit.php?post_type=give_forms&page=give-donors';
```

Comment:
```suggestion
                        window.location.href = '<?php echo admin_url('edit.php?post_type=give_forms&page=give-donors'); ?>';
```

Reply:
Being that this is PHP we can inject the generated admin URL using WordPress functions.

Reply:
You're right, it's better. We can also do that in DonationForms. I copied the code.